tests/test_magnet_coils.py

```diff
@@ -2,42 +2,95 @@
 
 import pytest
 import numpy as np
+import cadquery as cq
 
 import parastell.magnet_coils as magnet_coils
 from parastell.cubit_io import create_new_cubit_instance
 
+files_to_remove = [
+    "magnet_set.step",
+    "magnet_mesh.exo",
+    "magnet_mesh.h5m",
+    "stellarator.log",
+    "step_import.log",
+    "single_coil.step",
+]
+
 
 def remove_files():
+    for file in files_to_remove:
+        if Path(file).exists():
+            Path.unlink(file)
+
```

Comment:
Nice

Reply:
I'd like to change all of the tests to use this routine. I think a separate PR would be most appropriate for that though.